dist/placeholder-buster.txt

```diff
@@ -346,6 +360,12 @@ fearby.com###custom_html-22 > .widget-wrap.widget_text
 # https://github.com/NanoAdblockerLab/NanoContrib/pull/15#issuecomment-473121542
 thesimsresource.com##.crtv-top-wrapper
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+forbes.com##.adblock-unused
+
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/17
+howto-connect.com##.widget_text_smart:has(script[src*="adsbygoogle.js"])
```

Comment:
Why not check for `ins.adsbygoogle`? Seems more robust.

Reply:
Ah, thanks for the tip. 👍